### ISSUE TITLE
change max parking duration and opening times in parking overlay

### ISSUE THREAD

**Issue by Mannshoch** [streetcomplete/StreetComplete#5978]
### General
On many parking lots you have limits for parking duration and sometimes Opening times.

Please ad an edit for Parking in Parking lot Overlay
Where I could add or change a max parking allowed time and opening time.

**Comment by westnordost** [comment]
No. The overlays display and let you change one thing and one thing only. In respect to the parking overlay, it is exclusively about the physical parking on roads, nothing else.
